ext/page_log/palm/palm.c

```diff
@@ -418,17 +415,22 @@ static int
 palm_begin_checkpoint(WT_PAGE_LOG *page_log, WT_SESSION *session, uint64_t checkpoint_id)
 {
     PALM *palm;
+    PALM_KV_CONTEXT context;
```

Comment:
This ended up being a bunch of work I didn't anticipate doing, but it does enable a lot more interesting testing scenarios around restarts so I think it's worth the investment.